docs/tutorials/hello-nais.md

```diff
@@ -141,7 +145,7 @@ Add the following content to the file, and insert the appropriate values in the
         - name: Deploy to NAIS
           uses: nais/deploy/actions/deploy@v2
           env:
-            CLUSTER: <MY_CLUSTER> # Replace
+            CLUSTER: <MY_CLUSTER> # Replace. Valid environments are dev-gcp or prod-gcp
```

Comment:
Nais brukes ikke bare i NAV, så det er misvisende å skrive `dev-gcp` og `prod-gcp` her. Vi prøver å holde dokumentasjonen vår nøytral. Har en egen katalog for NAV-spesifikk dokumentasjon.

Reply:
Ja, mulig det ikke hører hjemme her. Tar vekk setningen igjen. Motivert av at jeg ikke fant beskrivelse av miljøene noe sted :)

Reply:
Her kan vi heller ha en lenke til miljøene som finnes. De er tenant-spesifikke. Kan få fikset det i morgen.
https://docs.nais.io/workloads/reference/environments/